src/taskpane/taskpane.js

```diff
@@ -1224,11 +1316,13 @@ function errorPopUp(type, err) {
     console.log(err);
```

Comment:
You need to add the same line as for Excel here: "artifactUpdateUI(4);" (corrected with comment and no magic number)